dockerfiles/Dockerfile.android-builder-base

```diff
@@ -0,0 +1,28 @@
+# TODO: Candidate for moving to its own repo for sharing across notifier repositories
+FROM openjdk:8-jdk-stretch
+
+# OS setup
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y gradle jq git-core build-essential openssl libssl-dev
+
+# Android tools
+WORKDIR /sdk
+RUN wget https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip -q \
+    && unzip -q sdk-tools-linux-4333796.zip \
+    && rm sdk-tools-linux-4333796.zip
+
+ENV PATH="${PATH}:/sdk/tools:/sdk/tools/bin"
+
+RUN yes | sdkmanager "platform-tools" "build-tools;28.0.3"
+ENV PATH="${PATH}:/sdk/platform-tools"
+
+ENV ANDROID_HOME="/sdk/"
+ENV GRADLE_OPTS="-Dorg.gradle.daemon=false"
+
+# Pre-download Gradle
+ENV GRADLE_USER_HOME="/app/gradle"
+WORKDIR /app/gradle
+
+RUN wget -q https://services.gradle.org/distributions/gradle-5.1.1-all.zip \
```

Comment:
are these the versions known statically? Or will this get out of date when we add a new RN fixture, and (potentially) unknowingly cause a different version of gradle to be downloaded on-demand way later in the build?

Reply:
You're right, they could get out of date and cause a download later in the build.  It's not a massive issue on CI as it's so fast out there to download things, but it is an area to work on improving.